src/app/src/pages/redteam/setup/components/StrategyConfigDialog.tsx

```diff
@@ -29,13 +39,17 @@ const DEFAULT_SIMBA_GOALS = [
   'Manipulate order status or pricing through injection attacks',
 ];
 
+// Strategies that can be used in layer steps (exclude layer itself to prevent recursion)
+const AVAILABLE_LAYER_STRATEGIES = ADDITIONAL_STRATEGIES.filter((s) => s !== 'layer').sort();
```

Comment:
thought: why are all of our strategies in a variable called `ADDITIONAL_STRATEGIES`?